x-pack/plugins/ingest_manager/server/services/epm/kibana/index_pattern/install.ts

```diff
@@ -95,26 +94,32 @@ export async function installIndexPatterns(
   );
   await Promise.all(packagePromises);
 
+  const packageVersionsToFetch = [...installedPackages];
   if (pkgName && pkgVersion) {
-    // add this package to the array if it doesn't already exist
-    const foundPkg = installedPackages.find((pkg) => pkg.pkgName === pkgName);
-    // this may be removed if we add the packged to saved objects before installing index patterns
-    // otherwise this is a first time install
-    // TODO: handle update case when versions are different
-    if (!foundPkg) {
-      installedPackages.push({ pkgName, pkgVersion });
+    const packageToInstall = packageVersionsToFetch.findIndex((pkg) => pkg.pkgName === pkgName);
```

Comment:
nitpick: any reason of not using find here? and mutate `packageToInstall. pkgVersion `  after?

Reply:
Thanks, I'll update 👍